rust-sdk/whirlpool/src/tests/utils/rpc.rs

```diff
@@ -188,6 +213,66 @@ impl RpcContext {
         let signature = self.rpc.send_transaction(&transaction).await?;
         Ok(signature)
     }
+
+    pub async fn create_token_2022_mint(
```

Comment:
Might be nice to put this function in a separate `token_extensions` file (just like we have the `token` file). If we add everything here this file will become very bloated

Reply:
Yep, I think that's more native rust way, just tried to follow ts-sdk structure but I can fix